src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.todo-container {
+  width: 300px;
+  margin-left: 30%;
+  box-shadow: 5px 5px 5px #b8b8b8;
+}
+
+.item li {
+  display: grid;
+  grid-template-columns: 10% 85% 5%;
+}
+
+i {
+  cursor: pointer;
+}
+
+.item li i {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.todo-input {
+  width: 100%;
+}
+
+.check {
+  text-decoration: line-through;
+  color: #b8b8b8;
+}
+
+#delete {
+  display: none;
+}
```

Comment:
I would kindly request to remove this unused code as it makes the code look disorganized